src/Model/Entity/IStorable.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Friendica\Model\Entity;
+
+/**
+ * @property int id
+ */
+interface IStorable
```

Comment:
This interface is necessary to use "abstract" Repository features (select, delete, save).
Each entity, which is persistent to the database must implement this interface.